index.html

```diff
@@ -18,14 +18,14 @@
     <br /><br />
     <p id='part2'>
         Avec votre patrimoine de
-        <input id='yourNetWorth' type='number' value='117000' autocomplete='off'>&nbsp;€,
+        <input id='yourNetWorth' type='text' value='0' autocomplete='off'>&nbsp;€,
         <br />
         c'est comme si vous donniez
         <br /><br />
-        <span id='donationAmount'>5,57&nbsp;€</span>
+        <span id='donationAmount'></span>
     </p>
     <p id='eqLine'>
-        Ce qui équivaut <span id='equivalent'>à un kebab</span>.
+        Ce qui équivaut <span id='equivalent'></span>.
```

Comment:
Users who disable JavaScript in their browsers won't be able to view the data, but it's the most effective way to ensure proper data maintenance.